lua/ruby_nvim/cmd.lua

```diff
@@ -89,4 +83,31 @@ M.test = function(current_line)
   run_in_floating_window(opts.load("test_cmd"), args)
 end
 
+-- :RubyBrowseGem
+
+M.browse_gem = function()
+  local line = api.nvim_get_current_line()
+  local gem_word = 'gem'
+  local gem_index = line:find(gem_word)
+  local gem_with_space_and_quote_length = ('gem' .. ' "'):len()
+  local quote_and_comma_length = ('",'):len()
```

Comment:
I think it is not necessary to create strings to count chars, these are fixed numbers:

```suggestion
  local gem_with_space_and_quote_length = 5
  local quote_and_comma_length = 2
```